### PR TITLE
Refactor & update with new metadata and changed API 

### PR DESCRIPTION
**Pull request by stephhazlitt**
- refactors `get-data.R` script to read in JSON metadata files
- tidies json and csv APIs 
- changes variables to coincide to API changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,21 @@
...
+The dip-metadata-viewer is built using the [{flexdashboard}](https://rmarkdown.rstudio.com/flexdashboard/), [{crosstalk}](https://rstudio.github.io/crosstalk/index.html)  and [{reactable}](https://glin.github.io/reactable/) R packages. All metadata is sourced from the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/group/data-innovation-program) using the [{bcdata}](https://bcgov.github.io/bcdata/) R package. All the data described in the dip-metadata-viewer is licensed [Access Only](https://www2.gov.bc.ca/gov/content?id=1AAACC9C65754E4D89A118B875E0FBDA) and is only available through the [Data Innovation Program](https://www2.gov.bc.ca/gov/content/data/about-data-management/data-innovation-program)&mdash;the data is _not_ downloadable from the B.C. Data Catalogue.
 
+### Code
+There are two scripts that are required to generate the viewer, they need to be run in order:
+
+- `get-data.R`
+- `dip-metadata-viewer.Rmd`
```
Comment (lindsay-forestell) [inline README.md:14]: For the non-heavy R user, it's more than just _running_ the Rmd file right - you have to knit it to an html?

File: get-data.R [hunk 2]
```diff
@@ -1,209 +1,175 @@
...
-tidy_metadata <- df_metadata_reduced %>%
-  mutate(
-    identifier_classification = str_replace(identifier_classification, "�", "-"),
-    field_description = str_replace_all(field_description, "�", " ")
-  ) %>%
+## Join JSON+CSV tables and final tidying
+tidy_metadata <- df_metadata_csv_reduced |> 
+  bind_rows(df_metadata_json) |> 
   mutate(
-    data_provider = case_when(
```
Comment (lindsay-forestell) [inline get-data.R:168]: It's a shame this field is no longer useful! And a quick look at the catalogue doesn't show a different proxy that can be accessed.

**Review by lindsay-fredrick — APPROVED** [review]
This ran and reproduced the html file nicely for me after installing all the packages I didn't have yet. My comments are only regarding the decline in amount of data available in the records now - the code itself looks fine, and I don't see any glaring methods of speeding it up!

File: README.md (hunk 1 above)
Comment (stephhazlitt) [reply README.md:14]: Yes, good catch. I'll edit the `README` to be more specific.

**Pull request merged**